Util/Fixtures/Customer/AddressBuilder.php

```diff
@@ -0,0 +1,260 @@
+    {
+        $builder = clone $this;
+        $builder->address->setFirstname($firstname);
+        return $builder;
+    }
+
+    public function withMiddlename(string $middlename): AddressBuilder
+    {
+        $builder = clone $this;
+        $builder->address->setMiddlename($middlename);
+        return $builder;
+    }
+
+    public function withLastname(string $lastname): AddressBuilder
+    {
+        $builder = clone $this;
+        $builder->address->setLastname($lastname);
+        return $builder;
+    }
+
+    public function withSuffix(string $suffix): AddressBuilder
+    {
+        $builder = clone $this;
+        $builder->address->setSuffix($suffix);
+        return $builder;
+    }
+
+    public function withStreet(string $street): AddressBuilder
+    {
+        $builder = clone $this;
+        $builder->address->setStreet((array)$street);
+        return $builder;
+    }
+
+    public function withCompany(string $company): AddressBuilder
+    {
+        $builder = clone $this;
+        $builder->address->setCompany($company);
+        return $builder;
+    }
+
+    public function withTelephone(string $telephone): AddressBuilder
+    {
+        $builder = clone $this;
+        $builder->address->setTelephone($telephone);
+        return $builder;
+    }
+
+    public function withPostcode(string $postcode): AddressBuilder
+    {
+        $builder = clone $this;
+        $builder->address->setPostcode($postcode);
+        return $builder;
+    }
+
+    public function withCity(string $city): AddressBuilder
+    {
+        $builder = clone $this;
+        $builder->address->setCity($city);
+        return $builder;
+    }
+
+    public function withCountryId(string $countryId): AddressBuilder
+    {
+        $builder = clone $this;
+        $builder->address->setCountryId($countryId);
+        return $builder;
+    }
+
+    public function withRegionId(int $regionId): AddressBuilder
+    {
+        $builder = clone $this;
+        $builder->address->setRegionId($regionId);
+        return $builder;
+    }
+
+    /**
+     * @param mixed[] $values
+     * @return AddressBuilder
+     */
+    public function withCustomAttributes(array $values): AddressBuilder
+    {
+        $builder = clone $this;
+        foreach ($values as $code => $value) {
+            $builder->address->setCustomAttribute($code, $value);
+        }
+        return $builder;
+    }
+
+    /**
+     * @return AddressInterface
+     * @throws LocalizedException
+     */
+    public function build(): AddressInterface
+    {
+        return $this->addressRepository->save($this->address);
+    }
+
+    public function buildWithoutSave(): AddressInterface
+    {
+        return clone $this->address;
+    }
+
+    private static function prepareFakeAddress(
+        ObjectManagerInterface $objectManager,
+        string $locale = 'en_US'
+    ): AddressInterface {
+        $faker = self::getAddressObject($locale);
```

Comment:
Perhaps since we have removed faker as a dependency we should change the name of this variable. If I read this code I would assume the dependency was there. Maybe just fakeAddress would work?

Reply:
Good catch